libnavigation-ui/src/test/java/com/mapbox/navigation/ui/camera/NavigationCameraTest.java

```diff
@@ -184,6 +191,62 @@ public void update_overrideSetsLocationComponentCameraModeNone() {
     verify(locationComponent).setCameraMode(eq(CameraMode.NONE));
   }
 
+  @Test
+  public void onShowRouteOverview_routeProgressAndRouteIsNull() {
+    MapboxMap mapboxMap = mock(MapboxMap.class);
+    MapboxNavigation navigation = mock(MapboxNavigation.class);
+    NavigationCamera camera = buildCamera(mapboxMap, navigation);
+
+    camera.start(null);
+    camera.showRouteOverview(new int[4]);
+
+    // verify no crash
+  }
+
+  @Test
+  public void onShowRouteOverview_routeProgressIsNull() {
```

Comment:
NIT

```suggestion
  public void onShowRouteOverview_whenRouteProgressIsNull() {
```

Maybe clearer? 🤔 

```suggestion
  public void animateCameraIsCalledWhenShowRouteOverviewIfRouteProgressIsNull() {
```